molecule/command/init/role.py

```diff
@@ -72,12 +75,10 @@ def execute(self):
             )
             util.sysexit_with_message(msg)
 
-        template_directory = ''
-        if 'template' in self._command_args.keys():
-            template_directory = self._command_args['template']
-        else:
-            template_directory = 'role'
-        self._process_templates(template_directory, self._command_args, role_directory)
+        r = subprocess.run(["ansible-galaxy", "init", role_name], shell=False)
```

Comment:
Shouldn't we use `check_output` or? The error message below will not be enough to help users when it fails.

Reply:
run() does not hide output, so there is no need to capture and display what was already logged.

Reply:
In a stack trace? We should handle that with an exception and pass to our logger?